useraccount/internal/adapters/framework/driver/grpc/proto/server.go

```diff
@@ -0,0 +1,18 @@
+package rpc
+
+
+type Adapter struct {
+
```

Comment:
A bit confused about this part in the video. He seems to add ports.APIPort struct element here. We can jump on a zoom call to discuss it further

Reply:
Yeah,  we can add that once we have one